### ISSUE TITLE
Document Category is unclear

### ISSUE THREAD

**Issue by elear** [secvisogram/secvisogram#205]
Hi everyone,

The "Document Category" field lists several types of CSAF documents.  The problem is that it is unclear to the user as to what those fields mean.  Because it is difficult to add helper text to a drop down, my suggestion is that you turn this element into a radial, and then add helper text that can have a brief explanation and include links to additional information.

**Comment by sthagen** [comment]
I thought the document categories were chosen as to speak for themselves - in case they do not, one can easily use the label and amend some hint - the value is not coupled to the label (even in plain old html), right? No need to change the interface element IMO.

**Comment by elear** [comment]
The reason I opened the issue was that I'm quite certain few people know what a "vex" is (**I** know what a vex is, but I doubt I am in the majority), and I don't know that people will understand when to use that versus a security_advisory versus an information_advisory versus...

**Comment by sthagen** [comment]
@elear the semantics of the categories have to be clear to typical users (a role I am not in)  and I support any enhancement to ease of use, as long as we do not 

* change interface elements for adding secondary information :+1: 
  * the former should be a given from interface design.
* embed the complete specification prose into the interface

The OASIS CSAF TC publishes the CSAF v2.0 as JSON Schema files. So, application developers may expect to extract or derive any descriptive (secondary) content directly from corresponding schema annotations (per title, description, examples, default, deprecated, readOnly, and writeOnly).

For enumeration values this is hard to provide per schema. In the prose the current CS01 provides in section 3.2.1.3 _Document Property - Category_ the following (hacked up highlight per diff block added for emphasis by me):

Document category (`category`) with value type `string` of 1 or more characters defines 
```diff 
+ a short canonical name, chosen by the document producer, 
+ which will inform the end user as to the category of document.
``` 

>It is directly related to the profiles defined in section 4.

(End of "citation")

**Comment by elear** [comment]
Yeah, I'm just thinking of some onmouseover() text on a radial instead of a pulldown...  No spec change, tho perhaps a link to a reference.

**Comment by tschmidtb51** [comment]
@elear: We plan to add a Wizard to assist the user in generating CSAF documents (including VEXs). Obviously, it won't be added this year. However, we are currently looking at some tenders already.

**Comment by tschmidtb51** [comment]
Next year, we will develop and add a wizard which provides guidance on how to create CSAF documents.

**Comment by tschmidtb51** [comment]
We just release [Secvisogram 2.1.0](https://secvisogram.github.io/) which has a sidebar that provides context-specific usage for the fields.

**Comment by tschmidtb51** [comment]
This issue was solved with Secvisogram 2.1.0
